spec/lib/que/worker_spec.rb

```diff
@@ -60,8 +60,8 @@
         end
       end
 
-      context "if postgres raises an exception while processing a job" do
-        it "rescues it and returns error" do
+      context "when postgres raises an exception while processing a job" do
+        it "we rescue it and return error" do
```

Comment:
So now the contexts are 👌 but the "it"s aren't! We can usually read these as a sentence, but "it we rescue it and return error" doesn't fly. The previous wording seemed good?